.github/workflows/test_compare_stacks.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       matrix:
         EESSI_VERSION:
         - 2023.06
+        - 2025.06
```

Comment:
Are we ignoring A64FX for this scenario? (It is likely to be a bit of a problem case, so I'm fine with that, but it is also nice to be able to see how it is not consistent)